docs/pages/admin-guides/management/admin/daemon.mdx

```diff
@@ -9,7 +9,15 @@ On Linux systems in non-containerized environments, we recommend running the `te
 Using a daemon helps to ensure that the `teleport` process can remain running and available regardless of a controlling terminal session or parent process, and opens up additional
 configuration options that allow for better optimization, uptime, and availability.
 
-This guide will outline best practices for installing, configuring, and starting teleport as a daemon on a Linux host using systemd.
+This guide will outline best practices for installing, configuring, and starting `teleport` as a daemon on a Linux host using systemd.
```

Comment:
Question - do we believe that this guide still covers the best practices for running Teleport on a Linux host?

I wonder if we think it's time to reflect that using `teleport-update` is considered best practice.

@hugoShaka knows best here.

Reply:
Thank you @zmb3, yes `teleport-update` now takes care of installing and setting up the system service, this is how we prevent users from making mistakes. The recommended install should now be:
- run `curl https://teleport.example.com/scripts/install.sh | sudo bash`
- run `teleport configure`
- run `systemctl enable teleport`